backend/src/models/projects/budget.js

```diff
@@ -211,6 +211,22 @@ const removeOne = (projectId) => {
   return knex(`${projectBudgetTable} as prb`).where("prb.id", projectId).del();
 };
 
+const findRecoveredBudgets = (quarter, projectId, fiscal, portfolio) => {
+  return knex
+    .select(
+      knex.raw(`
```

Comment:
are you sure you need to do raw here? i would expected that sum would be part of knex